src/gondi.ts

```diff
@@ -1067,6 +1073,121 @@ export class Gondi {
   } & Parameters<Contracts['UserVaultV5']['burnAndWithdraw']>[0]) {
     return this.contracts.UserVault(userVaultAddress).burnAndWithdraw(data);
   }
+  async wrapOldERC721({
+    collection,
+    tokenId,
+  }: {
+    collection: {
+      id: string;
+      wrapperCollections?: { contractData: { contractAddress: Address } }[];
+    };
+    tokenId: bigint;
+  }) {
+    if (collection?.id === undefined) throw Error('Invalid collection');
+
+    const wrapperAddress = await this._getWrapperAddress(collection);
```

Comment:
Hm. I don't think we should do this here (also talking about method definition). Let's give it some thought